common/sass/styles/overrides/react-select.scss

```diff
@@ -0,0 +1,62 @@
+// This syntax is necessary to override css styles in react-select
+.Select {
+  font-size: 1rem;
+  .Select-control {
+    box-sizing: content-box;
+    font-weight: 400;
+    border-radius: 0px;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
```

Comment:
We should make all of the instances of `rgba(0, 0, 0, 0.075)` and `rgba(0, 0, 0, 0.125)` variables, something like `$input-base-shadow` and `$input-base-shadow-focused` or whatever. They get repeated a lot.